README.md

```diff
@@ -17,8 +17,6 @@
 1. FingerprintJS Pro Google Tag Manager template
 2. FingerprintJS Pro adapter for Google Tag Manager
```

Comment:
Who would this be useful for? It seems like an implementation detail from my POV, considering that this integration is mostly aimed at less technical users. Have you considered putting it inside a separate README, or making it less prominent?